op-supervisor/supervisor/backend/backend.go

```diff
@@ -36,13 +35,22 @@ type SupervisorBackend struct {
 	// Write = set of chains is changing.
 	mu sync.RWMutex
 
+	// depSet is the dependency set that the backend uses to know about the chains it is indexing
 	depSet depset.DependencySet
 
-	// db holds on to the DB indices for each chain
-	db *db.ChainsDB
+	// chainDBs holds on to the DB indices for each chain
+	chainDBs *db.ChainsDB
 
 	// chainProcessors are notified of new unsafe blocks, and add the unsafe log events data into the events DB
 	chainProcessors map[types.ChainID]*processors.ChainProcessor
+
+	// synchronousProcessors disables background-workers,
+	// requiring manual triggers for the backend to process anything.
+	synchronousProcessors bool
+
+	// chainMetrics are used to track metrics for each chain
+	// they are reused for processors and databases of the same chain
+	chainMetrics map[types.ChainID]*chainMetrics
```

Comment:
chainMetrics are now held by the Backend so they may be referenced later by resources which get spun up later.